app/build.gradle

```diff
@@ -77,11 +95,10 @@ dependencies {
     implementation "com.google.dagger:hilt-android:2.44"
     kapt "com.google.dagger:hilt-compiler:2.44"
     // Navigation graph
-    implementation"androidx.navigation:navigation-fragment-ktx:2.5.3"
-    implementation"androidx.navigation:navigation-ui-ktx:2.5.3"
+    implementation "androidx.navigation:navigation-fragment-ktx:2.5.3"
+    implementation "androidx.navigation:navigation-ui-ktx:2.5.3"
```

Comment:
The best way to extract versions to another place, as Eugenius mentioned yesterday